code/modules/antagonists/antag_org/antag_org_datum.dm

```diff
@@ -0,0 +1,16 @@
+ /**
+  * The lore organisation antagonists are attached to. Influences objectives and steal targets.
+  */
+/datum/antag_org
+	/// Organisation's name
+	var/name = "Generic Bad Guys"
+	/// Description given to the antagonist on spawn, below 'You are a Traitor!' or similar
+	var/intro_desc = "You are not meant to see this. Please tell admins/coders that the antag_org wasn't set properly."
+	/// Used for prob() for objectives. Higher focus means the org is less likely to diverge from their favorites.
+	var/focus = 100
+	/// If set, the antag's first objective will be forced to this.
+	var/datum/objective/forced_objective
```

Comment:
might this be worth being a list? incase we want to force multiple objectives.

Reply:
Making this a list wouldn't be the worst thing, I agree